src/Microsoft.AspNet.SignalR.Core/Configuration/DefaultConfigurationManager.cs

```diff
@@ -14,12 +14,15 @@ public class DefaultConfigurationManager : IConfigurationManager
         // if _minimumKeepAlivesPerDisconnectTimeout != 3, update the ArguementOutOfRanceExceptionMessage below
         private const int _minimumKeepAlivesPerDisconnectTimeout = 3;
 
+        internal const int DefaultMaxScaleoutMappingsPerStream = 1000000;
```

Comment:
I don't think we should worry too much about compatibility here. Especially now that it's configurable.

Before we were extremely conservative with the max scaleout mappings in terms of making sure we wouldn't lose messages during reconnects. I remember the conversations back then and they basically amounted to "memory is cheap." As we've discovered, in many cases it's not.